Show STATUSMSG indication for actions as well

supertassu opened the pull request:
It is possible to send actions (/me et al.) to a STATUSMSG target (+#channel etc), even though thelounge does not currently support sending that. Show the statusmsg indicator on such messages.

supertassu committed:
Show STATUSMSG indication for actions as well

It is possible to send actions (/me et al.) to a STATUSMSG target
(+#channel etc), even though thelounge currently does not support
sending those. Show the statusmsg indicator on such messages.

brunnre8 reviewed (changes requested):
Thanks, can you reduce the dependency of the thing? There's no reason that the new component takes a message

supertassu commented on client/components/StatusmsgMarker.vue:
```diff
@@ -0,0 +1,21 @@
+export default defineComponent({
+	name: "StatusmsgMarker",
+	props: {
+		message: {type: Object as PropType<ClientMessage>, required: true},
```
good point, done!

MaxLeiter commented on client/components/StatusmsgMarker.vue:
```diff
@@ -0,0 +1,19 @@
+export default defineComponent({
+	name: "StatusmsgMarker",
+	props: {
+		group: {type: String, required: true},
```
```suggestion
		group: {type: String, required: false, default: undefined},
```

I believe this is More Correct, as `message.statusmsgGroup` can be `undefined` when its not a STATUSMSG.

supertassu replied:
Oh, indeed. Fixed, and fixed the typing of the `Msg` where I originally got this from.

supertassu committed:
server: models: Allow statusmsgGroup to be undefined

This will in practice be undefined when the message is not using
STATUSMSG, fix the type accordingly.
